modules/nf-core/multivcfanalyzer/tests/main.nf.test

```diff
@@ -0,0 +1,124 @@
+nextflow_process {
+
+    name "Test Process MULTIVCFANALYZER"
+    script "../main.nf"
+    process "MULTIVCFANALYZER"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "multivcfanalyzer"
+
+    setup {
+            run("GATK_UNIFIEDGENOTYPER") {
+                script "../../gatk/unifiedgenotyper/main.nf"
+                process{
+                    """
+                    input[0] = Channel.of([ [ id:'test' ], // meta map
+                           file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                           file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true)
+                         ],
+                         [ [ id:'test2' ], // meta map
+                           file(params.test_data['sarscov2']['illumina']['test_single_end_sorted_bam'], checkIfExists: true),
+                           file(params.test_data['sarscov2']['illumina']['test_single_end_sorted_bam_bai'], checkIfExists: true)
+                         ]
+                         )
+                    input[1] = [ [],
+                        file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                        ]
+                    input[2] = [ [],
+                        file(params.test_data['sarscov2']['genome']['genome_fasta_fai'], checkIfExists: true)
+                        ]
+                    input[3] = [ [],
+                        file(params.test_data['sarscov2']['genome']['genome_dict'], checkIfExists: true)
+                        ]
+                    input[4] = [[],[]]
+                    input[5] = [[],[]]
+                    input[6] = [[],[]]
+                    input[7] = [[],[]]
+                    """
+                }
+            }
+            run("GUNZIP") {
+                script "../../gunzip/main.nf"
+                process{
+                    """
+                    input[0] = GATK_UNIFIEDGENOTYPER.out.vcf
+                    """
+                }
+            }
+        }
+
+    test("sarscov2 - vcf") {
+
+        when {
+            process {
+                """
+                input[0] = GUNZIP.out.gunzip.collect{ meta, vcf -> vcf }.map{ vcf -> [[ id: 'testVCF'], vcf]}
```

Comment:
```suggestion
                input[0] = GUNZIP.out.gunzip
```

Is that necessary? You already have a meta, vcf, why do you remove it and add it again?

If you accept my change, make sure to update the meta.id in the GATK UnifiedGenotyper to `testVCF` and you hopefully won't need to rebuild the snapshot (as the resulting name should be the same)

Reply:
I think this is necessary because we want all the vcf listed under the same meta, so we can do the vcfs.join(" ") in the main.nf. I tried removing it and putting it on the GATK, but then there is only a single vcf file.

Reply:
Yes, my bad misread the code